src/list-data/entity-type-data.ts

```diff
@@ -31,6 +31,7 @@ export const EntityTypesBcData: EntityI[] = [
   },
   {
     text: 'Limited Company',
+    value: EntityType.CR,
```

Comment:
In this file, I moved the value property to the top of each record to be consistent and to more easily see what type each record is.